MantidQt/MantidWidgets/src/DataProcessorUI/GenericDataProcessorPresenter.cpp

```diff
@@ -1409,20 +1442,18 @@ void GenericDataProcessorPresenter::plotGroup() {
     throw std::runtime_error("Can't plot group.");
 
   // Set of workspaces to plot
-  std::set<std::string> workspaces;
+  std::set<QString> workspaces;
   // Set of workspaces not found in the ADS
-  std::set<std::string> notFound;
+  QSet<QString> notFound;
 
-  const auto items = m_manager->selectedData();
+  auto const items = m_manager->selectedData();
```

Comment:
That's a good idea. The codebase doesn't seem to be all that consistent with regard to the position of the const. 

I tend to prefer `const` on the right as to me it seems more consistent and easy to remember which part is constant, especially in situations like `int const *` where if you're used to `const` applying to the thing on the right you may end up thinking it's the pointer which is constant rather than the pointee. Additionally methods which are `const` can only have `const` on the right.

> const applies to the thing on the left

is easier to remember than

> const applies to the thing on the left except when it doesn't or can't in which case it applies to the thing on the right.

Regardless it would be good to adopt one style or the other.